src/SpecificDissipationRateEquationSystem.C

```diff
@@ -408,10 +423,27 @@ SpecificDissipationRateEquationSystem::register_interior_algorithm(
   std::map<AlgorithmType, Algorithm *>::iterator itev =
     diffFluxCoeffAlgDriver_->algMap_.find(algType);
   if ( itev == diffFluxCoeffAlgDriver_->algMap_.end() ) {
-    const double sigmaWOne = realm_.get_turb_model_constant(TM_sigmaWOne);
-    const double sigmaWTwo = realm_.get_turb_model_constant(TM_sigmaWTwo);
-    EffectiveSSTDiffFluxCoeffAlgorithm *effDiffAlg
-      = new EffectiveSSTDiffFluxCoeffAlgorithm(realm_, part, visc_, tvisc_, evisc_, sigmaWOne, sigmaWTwo);
+      Algorithm *effDiffAlg = NULL;
+    switch(turbulenceModel_){
+    case SST: case SST_DES:	    
+    {
+     const double sigmaWOne = realm_.get_turb_model_constant(TM_sigmaWOne);
+     const double sigmaWTwo = realm_.get_turb_model_constant(TM_sigmaWTwo);
+     effDiffAlg  = new EffectiveSSTDiffFluxCoeffAlgorithm(realm_, part, visc_, tvisc_, evisc_, sigmaWOne, sigmaWTwo);
+    }
+    break;
+    case ABL_DES_BLEND:
+    {
+     const double lamSc  = realm_.get_lam_schmidt(tke_->name());
+     const double turbSc = realm_.get_lam_schmidt(tke_->name());
```

Comment:
should this be `realm_.get_turb_schmidt(tke_->name());`?  Also, this is the SDR Eq System file and you are using `tke_`, which I believe hasn't been defined at this point.  Is it the `tke_` quantities that you want here?

You seem to have some errors that would be caught during the build process, are you still having compilation errors?